db/migrate/20230605133804_remove_master_variants.rb

```diff
@@ -0,0 +1,40 @@
+class RemoveMasterVariants < ActiveRecord::Migration[7.0]
+  def change
+    if ActiveRecord::Base.connection.table_exists? "spree_option_values_variants"
```

Comment:
I was curious about this one, I'm guessing you have another branch in play that overlaps here?

Reply:
Yeah, that table isn't in use any more (`OptionValues` are gone now) and is in the process of being removed, but I wasn't sure if it would still be present or not at the exact time this PR was merged/deployed. So this migration will work correctly either way.

The point of interacting with that table here is that there's a foreign key constraint at the database level, so you can't delete a variant if there are associated records in the `spree_option_values_variants` table that match it's `id` (they would need to be deleted first). But if the table has been dropped already then it can be skipped.